MaterialDesignThemes.Wpf/Clock.cs

```diff
@@ -317,8 +317,13 @@ private void HourReadOutPartNameOnPreviewMouseLeftButtonDown(object sender, Mous
 				button.SetBinding(Canvas.LeftProperty, GetBinding("X", button));
 				button.SetBinding(Canvas.TopProperty, GetBinding("Y", button));
 
-				button.Content = (i == 60 ? 0 : (i == 24 ? 0 : i)).ToString(format);
-				canvas.Children.Add(button);
+			    if (Is24Hours) {
+			        button.Content = (i == 60 ? 0 : (i == 24 ? 0 : i)).ToString(format);
+			    }
+			    else {
```

Comment:
@mokeev1995 Er... Do you mind to tell me how to fix it? Because I'm not sure how to fix that